test/dlc_tests/benchmark/ec2/tensorflow/inference/test_performance_tensorflow_inference.py

```diff
@@ -97,7 +104,7 @@ def ec2_performance_tensorflow_inference(
     else:
         ec2_connection.run(f"pip3 install -U pip")
         ec2_connection.run(
-            f"pip3 install --user boto3 grpcio 'tensorflow-serving-api<={tf_api_version}'"
+            f"pip3 install --user --upgrade boto3 grpcio 'protobuf<3.21' 'numpy<1.24' 'tensorflow-serving-api<={tf_api_version}'"
```

Comment:
On this PR, did we happen to see benchmark failures on the non-graviton images? If not, we can probably revert this change and I can add the fix in non-graviton PR if required.

Reply:
I did see them as I was testing EC2 when I found the benchmark broken in the first place. will want to keep this for now.

Reply:
I was seeing errors in x86 when I locally tested, but will remove as you are working a PR for x86